### ISSUE TITLE
Part1-06 getting quite long

### ISSUE THREAD

**Issue by Leslie-C** [oneilsh/apcb_second_edition#9]
This chapter is getting quite long, especially if we add package manager info - @oneilsh, how would you feel about splitting it into 2?

I'm thinking a split right after the first section of exercises, still called "Installing (Bioinformatics) Software", and a second chapter maybe "Using (Bioinformatics) Software"? It would contain the headers:
- Getting Data
- Putting It All Together
- Reproducibility with Scripts

Thoughts?

**Comment by oneilsh** [comment]
Yeah, that's one of the three "apex" chapters in each part (mentioned [here](https://github.com/oneilsh/apcb_second_edition/blob/91c50b5d8bb19544dee8a07a91ed58a3bbb4ceb7/index.Rmd#L58) in the preface). Splitting it sounds good to me, perhaps the other corresponding ones need a similar treatment.

**Comment by Leslie-C** [comment]
Sounds good! I'll go ahead and split 1-06 now (since it affects numbering for the rest of p1), but leave this issue open so we remember to check in on the apex for P2-3, and so we remember to go back and edit preface.
